bin/butane.js

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env node
 
-var argv = require('minimist')(process.argv.slice(2));
-var fireRules = require('../dist/');
-
-var input = argv._[0];
-var output = argv._[1];
-
-fireRules.convert(input, output);
+require('require-relative-main')('./cli', __dirname)
```

Comment:
@bendrucker For now, to keep the dependencies small, would could you change this back to:

```javascript
require('../dist/cli');
```
and then remove the `require-relative-main` dependency?

We can always add it back if this pattern gets used often.